net/sshtunnel/files/sshtunnel.init

```diff
@@ -180,6 +181,11 @@ load_server() {
 
 	ARGS="$ARGS_options -o ExitOnForwardFailure=yes -o BatchMode=yes -nN $ARGS_tunnels -p $port $user@$hostname"
 
+	if [ ! -z "$ProxyCommand" ]; then
+		ARGS="$ARGS -o \"ProxyCommand ${ProxyCommand}\""
```

Comment:
! -z can be replaced with -n

Reply:
Hi @neheb ,

Thank you for the feedback!

> ! -z can be replaced with -n

Sure! I'll add this.
Futhermore, I've detected that it's necessary to use a temporal file to pass the proxy command because the spaces inside the command. I have a new version implemented that it's more robust. I'll publish it soon.

Regards.